adage/adage/config.py.template

```diff
@@ -7,6 +7,10 @@
 # and replace with values tailored to your deployment.
 import random
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Given the above, I don't think there is any way around having to *specify* the destination directory for the repo. That said, I don't see a reason we need to use an absolute path in that specification.

What we need here is a way to determine at deploy time where the `REPO_ROOT` will land, right? Is there a way we can reliably determine where that will be inside the Docker image? Does something fairly standard like `"$HOME"` or `~/` get us the right location? If so, we should switch back to using `home_dir` or define a `repo_root` in the `OS_CONFIG` parameters and I can tweak the `fab` commands to do the right thing.